lib/ViewModels/CompositeBar/CompositeBarItemController.ts

```diff
@@ -116,14 +116,14 @@ export abstract class CompositeBarItemController
   /**
    * Gets the {@link this._visible}
    */
-  @computed
   get visible(): boolean {
     return this._visible;
   }
 
-  @action
   setVisible(v: boolean) {
-    this._visible = v;
+    runInAction(() => {
```

Comment:
The method was previously decorated using `@action`. When child classes override this method, they'll have to remember to use `@override` - otherwise it causes an app crash. This is hard to enforce especially if community users have extended this class. By switching to `runInAction` instead, child classes can freely override the method.

Couple of things to note for the future: 
  - Mixing mobx decorators and inheritance is tricky.
  - Inheritance as an API mechanism is brittle because when overriding a method, the dependency between child class and parent class is implicit. This makes it difficult for upstream library to make changes without breaking downstream users.